h2/src/test/org/h2/test/server/TestAutoServer.java

```diff
@@ -72,42 +72,40 @@ private void testAutoServer(boolean port) throws Exception {
             url += ";AUTO_SERVER_PORT=11111";
         }
         String user = getUser(), password = getPassword();
-        Connection connServer = getConnection(url + ";OPEN_NEW=TRUE",
-                user, password);
-
-        int i = ITERATIONS;
-        for (; i > 0; i--) {
-            Thread.sleep(100);
-            SortedProperties prop = SortedProperties.loadProperties(
-                    getBaseDir() + "/" + getTestName() + ".lock.db");
-            String key = prop.getProperty("id");
-            String server = prop.getProperty("server");
-            if (server != null) {
-                String u2 = url.substring(url.indexOf(';'));
-                u2 = "jdbc:h2:tcp://" + server + "/" + key + u2;
-                Connection conn = DriverManager.getConnection(u2, user, password);
-                conn.close();
-                int gotPort = Integer.parseInt(server.substring(server.lastIndexOf(':') + 1));
-                if (port) {
-                    assertEquals(11111, gotPort);
+        Connection connServer = getConnection(url + ";OPEN_NEW=TRUE", user, password);
+            int i = ITERATIONS;
```

Comment:
It looks like indentation is too large here and below.

Reply:
It meant to be try-with-resorce